internal/cmd/mongodbflex/instance/create/create.go

```diff
@@ -100,6 +99,15 @@ func NewCmd() *cobra.Command {
 				}
 			}
 
+			// Fill in version, if needed
```

Comment:
Is there any specific reason this was not added to the `buildRequest` function instead? We are already also filling the flavor ID there